packages/datetime/test/dateInputTests.tsx

```diff
@@ -37,11 +37,20 @@ describe("<DateInput>", () => {
     });
 
     it("Popover doesn't open if disabled=true", () => {
-        const wrapper = mount(<DateInput disabled />);
-        wrapper.find(InputGroup).simulate("focus");
+        const wrapper = mount(<DateInput disabled={true} />);
+        wrapper.find("input").simulate("focus");
```

Comment:
this test didn't actually work before, now it does. (and i had to fix line 188 above)